bitcoin/src/network/constants.rs

```diff
@@ -239,7 +239,7 @@ impl From<Network> for Magic {
     }
 }
 
-/// Error in parsing magic from string.
+/// Error in creating a Network from Magic bytes.
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub struct UnknownMagic(Magic);
```

Comment:
OT: this struct is far away from its impls, we should move it below.

Reply:
What does OT stand for @Kixunil?

Reply:
Off topic.